index.html

```diff
@@ -13,73 +13,227 @@
-    <meta name="twitter:image" content="./assets/bh8/bh8_social_cards.png">
+    <meta name="twitter:image" content="./assets/social_cards.png">
 
     <!-- Open Graph Cards -->
     <meta property="og:type" content="website" />
     <meta property="og:title" content="Brickhack 8" />
     <meta property="og:site_name" content="BrickHack 8" />
     <meta property="og:url" content="https://brickhack.io" />
     <meta property="og:description" content="RIT's premiere collegiate hackathon devoted to bringing designers, developers, and all sorts of makers together for 24 hours to create something from nothing."/>
-    <meta property="og:image" content="./assets/bh8/bh8_social_cards.png" />
+    <meta property="og:image" content="./assets/social_cards.png" />
 
     <!-- Favicons -->
     <link rel="apple-touch-icon" sizes="180x180" href="resources/favicon/apple-touch-icon.png">
     <link rel="icon" type="image/png" sizes="32x32" href="resources/favicon/favicon-32x32.png">
     <link rel="icon" type="image/png" sizes="16x16" href="resources/favicon/favicon-16x16.png">
 
-    <!-- Custom Win98 stylesheet -->
-    <link rel="stylesheet" href="https://unpkg.com/98.css" />
+    <!-- Fonts -->
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Work+Sans:wght@400;600;700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Sora&display=swap" rel="stylesheet">
 </head>
 <body>
-    <section id="main">
+    <!-- Nav -->
+    <nav>
+        <div class="top">
+            <i id="toggle" class="fa fa-bars" aria-hidden="true" id="toggle"></i>
+            <a id="logo" href="https://brickhack.io">
+                <img src="assets/bh8/logo.svg" alt="BrickHack 8 Logo" />
+            </a>
+        </div>
+        <div class="left navbar-items">
+            <a class="link" href="#about">About</a>
+            <a class="link" href="#faq">FAQ</a>
+            <a class="link" href="#footer-content">Contact</a>
+        </div>
+        <div class="right">
+            <!-- Login and Signup links will go here -->
+        </div>
+    </nav>
+    <div class="mobile-grayout"></div>
+
+    <!-- MLH Banner -->
+    <a id="mlh-trust-badge" href="https://mlh.io/seasons/2022/events?utm_source=na-hackathon&utm_medium=TrustBadge&utm_campaign=2022-season&utm_content=gray" target="_blank">
+        <img src="https://s3.amazonaws.com/logged-assets/trust-badge/2022/mlh-trust-badge-2022-gray.svg" alt="Major League Hacking 2022 Hackathon Season" style="width:100%">
+    </a>
+
+    <!-- Hero -->
+    <section id="hero" class="section-pad-left">
 
-        <img id="logo" src="./resources/pixel-logo-real.png" alt="BrickHack 8 logo. Two stripes, left orange and right blue, in a parallelogram with a white border.">
+    </section>
+
+    <!-- About -->
+    <section id="about" class="section-pad-top">
```

Comment:
What about these classes? I don't think they exist in the CSS anymore
`(section-pad-top, section-pad-left, section-pad)`
(To be clear, agreed on their removal)

Reply:
They were removed

Forgot to mention that all content and direct styling of About, FAQ, and Contact were mostly left alone since they have their own issues to handle them later on. This PR is just the general styling of those sections, from typography to color to general section size/padding

Reply:
gotcha!